src/compiler/Stage/Emit/JavaScript.elm

```diff
@@ -4,7 +4,7 @@ module Stage.Emit.JavaScript exposing (emitExpr, emitTopLevelDeclaration)
 
 import AST.Backend as Backend
 import AST.Common.Literal exposing (Literal(..))
-import AST.Typed exposing (Expr_(..))
+import AST.Typed exposing (Expr_(..), getExpr)
```

Comment:
I'd rather not import `getExpr` like this but use it qualified (`AST.Typed.getExpr`)